src/plugins/unified_histogram/public/chart/breakdown_field_selector.tsx

```diff
@@ -107,3 +107,6 @@ export const BreakdownFieldSelector = ({
     />
   );
 };
+
+// eslint-disable-next-line import/no-default-export
+export default BreakdownFieldSelector;
```

Comment:
Maybe it's time to move it out to a new package so you don't have to depend on `unified-histogram` plugin. Are you targeting 8.14?

Reply:
Yeah, it'd be neat to not depend on `unified-histogram` only for `BreakdownFieldSelector `. Although there might be a possibility to use the whole histogram component in future, if in case more functionality is needed.

I am not particularly targeting `8.14` but would be great if it goes into it.

Where `BreakdownFieldSelector ` should be moved to, in a new package of its own under `packages/`or to some existing package, any ideas?

Reply:
To a new package under `packages/` or to `kbn-visualization-ui-components` if @elastic/kibana-visualizations is okay with it. It's also fine to do it later as a follow up PR.